cfme/v2v/migration_plans.py

```diff
@@ -0,0 +1,636 @@
+                self.title.text == 'Not Started Plans')
+
+
+class InProgressPlansView(MigrationPlanView):
+    progress_card = MigrationProgressBar()
+
+    @property
+    def is_displayed(self):
+            return (self.in_migration_plan and
+                    self.progress_card.is_displayed and
+                    self.title.text == 'In Progress Plans')
+
+
+class CompletedPlansView(MigrationPlanView):
+    plans_completed_list = MigrationPlansList("plans-complete-list")
+
+    @property
+    def is_displayed(self):
+            return (self.in_migration_plan and
+                    self.plans_completed_list.is_displayed and
+                    self.title.text == 'Completed Plans')
+
+
+class ArchivedPlansView(MigrationPlanView):
+    archived_plans_list = MigrationPlansList("plans-archived-list")
+
+    @property
+    def is_displayed(self):
+            return (self.in_migration_plan and
+                    self.archived_plans_list.is_displayed and
+                    self.title.text == 'Archived Plans')
+
+
+class AddMigrationPlanView(View):
+    title = Text(locator='.//h4[contains(@class,"modal-title")]')
+    back_btn = Button("Back")
+    next_btn = Button("Next")
+    cancel_btn = Button("Cancel")
+    close = Button("Close")
+    fill_strategy = WaitFillViewStrategy("15s")
+
+    @View.nested
+    class general(View):  # noqa
+        infra_map = BootstrapSelect("infrastructure_mapping")
+        name = TextInput(name="name")
+        name_help_text = Text(locator='.//div[contains(@id,"name")]/span')
+        description = TextInput(name="description")
+        select_vm = RadioGroup('.//div[contains(@id,"vm_choice_radio")]')
+        fill_strategy = WaitFillViewStrategy("15s")
+
+        @property
+        def is_displayed(self):
+            return self.infra_map.is_displayed and self.name.is_displayed
+
+        def after_fill(self, was_change):
+            self.parent.next_btn.click()
+
+    @View.nested
+    class vms(View):  # noqa
+        import_btn = Button("Import")
+        hidden_field = HiddenFileInput(locator='.//input[contains(@accept,".csv")]')
+
+        table = Table(
+            './/div[contains(@class, "container-fluid")]/table',
+            column_widgets={
+                "Select": Checkbox(locator=".//input"),
+                1: Text('.//span/button[contains(@class,"btn btn-link")]'),
+            },
+        )
+        filter_by_dropdown = SelectorDropdown("id", "filterFieldTypeMenu_vms_step")
+        search_box = TextInput(
+            locator='.//div[contains(@class, "modal-content")]'
+            '//div[contains(@class,"input-group")]/input'
+        )
+        clear_filters = Text(".//a[text()='Clear All Filters']")
+        error_text = Text(
+            './/h3[contains(@class,"blank-slate-pf-main-action") and '
+            'contains(text(), "Error:")]'
+        )
+        error_icon = Text(locator='.//span[contains(@class, "pficon-error-circle-o")]')
```

Comment:
This icon string is defined in wt.patternfly, it would be good to import and use it here.